tests/core/test_model.py

```diff
@@ -0,0 +1,95 @@
+"""Test module for model.py (and associated functionality).
+
+This module tests the functionality of model.py, as well as other bits of code that
+define models based on the class defined in model.py
```

Comment:
would you like to add here {MODEL_NAME}_model.py? Or mention that this is a general test for the model?

Reply:
Yeah that bit definitely could be clearer. I've tried to clarify that this script tests `model.py` which is the script that provides the framework that other specific models (e.g. `abiotic_model.py`) have to follow